### PR TITLE
Added additional preview cards with box cards.

### PR DESCRIPTION
**Pull request by clayton-grey**
Includes: Protecting the Arnirniq, Foresight, "Hit Me!", Gené Beauregard, Meditative Trance, Fire Extinguisher

### CODE REVIEW

File: pack/eoe/eoep.json [hunk 1]
```diff
@@ -472,6 +472,23 @@
...
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "guardian",
+        "illustrator": "Lindsey Messecar",
+        "name": "Geared Up",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 19,
+        "quantity": 1,
+        "subtype_code": "weakness",
```
Comment (robert-huang) [inline pack/eoe/eoep.json:486]: small mistake here, remove this line please

File: pack/eoe/eoep.json [hunk 2]
```diff
@@ -823,6 +840,40 @@
...
+        "illustrator": "Mauro Dal Bo",
+        "flavor": "You let your mind drift into cold, meaningless oblivion.\nThe emptiness feels like home.",
+        "name": "Meditative Trance",
+        "pack_code": "eoep",
+        "position": 61,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "skill_willpower": 1,
+        "text": "For each of your arcane slots that is filled, either heal 1 damage or 1 horror.",
+        "traits": "Inisght. Spirit.",
```
Comment (robert-huang) [inline pack/eoe/eoep.json:857]: typo Insight

File: pack/eoe/eoep.json [hunk 3]
```diff
@@ -890,6 +941,21 @@
...
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Marcin Jakubowski",
+        "name": "Fire Extinguisher",
+        "pack_code": "eoep",
+        "position": 80,
+        "quantity": 2,
+        "skill_agility": 2,
+        "slot": "Hand",
+        "text": "[action]: <b>Fight.</b> You get +1 [combat] and deal +1 damage for this attack.\n[action] Discard Fire Extinguisher: <b>Evade.</b> Automatically evade all enemies engaged with you. You may exile Fire Extinguisher instead of discarding it to discard each non-<b><i>Elite</i></b> evaded by this effect.",
```
Comment (robert-huang) [inline pack/eoe/eoep.json:955]: I believe we use [[Elite]] instead (and for anything that is a Trait in the card text)

File: pack/eoe/eoep.json [hunk 4]
```diff
@@ -1029,6 +1095,28 @@
         "type_code": "asset",
         "xp": 5
     },
+    {
+        "code": "08099",
+        "cost": 2,
```
Comment (robert-huang) [inline pack/eoe/eoep.json:1100]: cost 5

File: pack/eoe/eoep.json [hunk 5]
```diff
@@ -1029,6 +1095,28 @@
...
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "faction2_code": "rogue",
+        "health": 2,
+        "illustrator": "Vlad Ricean",
```
Comment (robert-huang) [inline pack/eoe/eoep.json:1105]: I don't believe we know for sure who the illustrator is right? Best I can find for this card is https://i.imgur.com/4vK8jyl_d.webp?maxwidth=760&fidelity=grand (and the unsanctioned spoiler has a camera flash hiding the illustrator name hahah)

File: pack/eoe/eoep.json [hunk 6]
```diff
@@ -1067,6 +1155,25 @@
...
+        "faction2_code": "mystic",
+        "flavor": "All living things have breath.\nBe careful they do not curse you with theirs.",
+        "illustrator": "Chris Pritchard",
+        "name": "Protecting the Anirniq",
+        "pack_code": "eoep",
+        "position": 102,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 1,
+        "text": "Fast. Play after an <b><i>Ally</i></b> asset at your location is discarded by a card effect or is defeated.\nEither return that asset to its owner's hand, or its owner draws 3 cards.",
```
Comment (robert-huang) [inline pack/eoe/eoep.json:1172]: [[Ally]]

**Review by robert-huang — CHANGES_REQUESTED** [review]

File: pack/eoe/eoep.json [hunk 7]
```diff
@@ -472,6 +472,23 @@
...
+        "deck_limit": 1,
+        "faction_code": "guardian",
+        "illustrator": "Lindsey Messecar",
+        "name": "Geared Up",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 19,
+        "quantity": 1,
+        "subtype_code": "weakness",
+        "text": "Permanent. Limit 1 per deck. Purchase at deck creation.\n<b>Forced</b> - When your first turn of the game begins: Play any number of <b><i>Item</i></b> assets from your hand, reducing the cost of each by 1. During this turn, you have 3 fewer actions to take.",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:487]: NB [[]] for traits in game text, not bold and ital tags.

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live - thanks

**Pull request merged**